src/script/conversation/ConversationFederationUtils.ts

```diff
@@ -59,3 +59,47 @@ export function processFederationDeleteEvent(
 
   return result;
 }
+
+export interface FederationConnectionRemovedResult {
+  conversationsToDeleteUsers: {conversation: Conversation; users: User[]}[];
+}
+
+export function processFederationConnectionRemovedEvent(
```

Comment:
suggestion: The naming is not really explicit to what it's actually computing. 
How about `getUsersToDeleteFromFederatedConversations`? 
WDYT?

Reply:
if we change `processFederationConnectionRemovedEvent` to `getUsersToDeleteFromFederatedConversations`
then we need to think of a name for `processFederationDeleteEvent` as well or we lose consistency of naming in this file...

Reply:
https://github.com/wireapp/wire-webapp/pull/15713/commits/82f7cd902d434976d9004dce3f015d629a59ca48